client/src/index.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from "react-router-dom";
 
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+    <Router>
+        <App />
+    </Router>, 
+document.getElementById('root'));
```

Comment:
It looks like this ought to be indented, and that the ending paren (and semicolon) ought to be moved to the next line. For example:
```javascript
ReactDOM.render(
  <Router>
    <App />
  </Router>, 
  document.getElementById('root')
);
```